### PR TITLE
Make accessUrl an optional argument

### PR DESCRIPTION
**Pull request by MagicalCodeMonkey**
This should make accessUrl optional. If not provided, it will use the information provided in the HTTP request to build the invite URL.

### CODE REVIEW

File: webapp.js [hunk 1]
```diff
@@ -91,8 +88,9 @@ const app = async (orm) => {
         msg: 'ERR: You did not send any POST data',
       }).end();
     }
+    console.log(`req.body:`, req.body);
```
Comment (samcm) [inline webapp.js:91]: I think we can drop this, right?

**Comment by samcm** [comment]
I think we can probably provide this value when creating an invite to be honest. If someone wants to override it (like when they're accessing via a local IP rather than an external) they can do so with the flag. What are your thoughts?

File: webapp.js (hunk 1 above)
Comment (MagicalCodeMonkey) [reply webapp.js:91]: Oops! Yeah, get rid of that garbage! 😆

**Comment by MagicalCodeMonkey** [comment]
Okay. I figured out what I was doing here! The urlOrigin is already supplied to the request but wasn't being used by waterline to set up the URL. Adding urlOrigin into the schema fixed that.
If you can think of a better way to do it, let me know!

**Comment by samcm** [comment]
❤️ I'll look at deploying this next release, thanks mate

**Pull request closed without merging**